### PR TITLE
Feature/implement schibsted cmp

### PR DESCRIPTION
**Pull request by pablovegau**
- Add SUI CMP component
- Add SUI lint
- change styled-components for scss
- change file structure

### CODE REVIEW

File: src/CMP_demo/user_interface/index.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans:400,700" rel="stylesheet">
+    <title>Boros.js Demo</title>
```
Comment (victuxbb) [inline src/CMP_demo/user_interface/index.html]: BorosJS without dot "."

Comment (victuxbb) [reply src/CMP_demo/user_interface/index.html]: well...also in capitalize mode JS

File: src/CMP_demo/user_interface/pages/methodPages/GetConsentData/GetConsentData.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+const baseClass = 'methodPages'
+
+class GetConsentData extends Component {
+  state = {
+    getConsentDataResponse: ''
+  }
+
+  handleGetConsentData = () => {
+    window.__cmp('getConsentData', null, result => {
```
Comment (victuxbb) [inline src/CMP_demo/user_interface/pages/methodPages/GetConsentData/GetConsentData.js]: result arrow function has 2 parameters, (data,success) =>

File: src/CMP_demo/user_interface/pages/methodPages/GetPublisherConsents/GetPublisherConsents.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+const baseClass = 'methodPages'
+
+class GetPublisherConsents extends Component {
+  state = {
+    getPublisherConsentsResponse: ''
+  }
+
+  handleGetPublisherConsents = () => {
+    window.__cmp('getPublisherConsents', null, result => {
```
Comment (victuxbb) [inline src/CMP_demo/user_interface/pages/methodPages/GetPublisherConsents/GetPublisherConsents.js]: same here...2 params arrow function

File: src/CMP_demo/user_interface/pages/methodPages/GetVendorConsents/GetVendorConsents.js [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+const baseClass = 'methodPages'
+
+class GetVendorConsents extends Component {
+  state = {
+    getVendorConsentsResponse: ''
+  }
+
+  handleGetVendorConsents = () => {
+    window.__cmp('getVendorConsents', null, result => {
```
Comment (victuxbb) [inline src/CMP_demo/user_interface/pages/methodPages/GetVendorConsents/GetVendorConsents.js]: same here

File: src/CMP_demo/user_interface/pages/methodPages/GetVendorList/GetVendorList.js [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+
+const baseClass = 'methodPages'
+
+class GetVendorList extends Component {
+  state = {
+    getVendorListResponse: ''
+  }
+
+  handleGetVendorList = () => {
+    window.__cmp('getVendorList', null, result => {
```
Comment (victuxbb) [inline src/CMP_demo/user_interface/pages/methodPages/GetVendorList/GetVendorList.js]: same here

File: src/CMP_demo/user_interface/pages/methodPages/Ping/Ping.js [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+
+const baseClass = 'methodPages'
+
+class Ping extends Component {
+  state = {
+    pingResponse: ''
+  }
+
+  handlePing = () => {
+    window.__cmp('ping', null, result => {
```
Comment (victuxbb) [inline src/CMP_demo/user_interface/pages/methodPages/Ping/Ping.js]: same here

**Comment by pablovegau** [comment]
Done! @victuxbb

**Pull request merged**